tide_core.install

```diff
@@ -1331,3 +1331,24 @@ function tide_core_update_8054() {
     $config->set('cardinality', -1)->save();
   }
 }
+
+/**
+ * Adds field_statistics_grid's storage
+ */
+function tide_core_update_8056() {
+  $configs = [
+    'field.storage.paragraph.field_statistics_grid_theme' => 'field_storage_config',
+    'field.storage.paragraph.field_statistic_heading' => 'field_storage_config',
+    'field.storage.paragraph.field_statistic_body' => 'field_storage_config',
+    'field.storage.paragraph.field_statistic_block' => 'field_storage_config',
+  ];
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_core') . '/config/install'];
+  // Check if field already exported to config/sync.
+  foreach ($configs as $config => $type) {
+    $config_read = _tide_read_config($config, $config_location, TRUE);
```

Comment:
hey @mayurngondhkar , please check the condition where the fields are installed.

Reply:
Changes made